packages/simulation/src/convert.ts

```diff
@@ -9,6 +9,14 @@ const makeStateLabel = (states: number[]): string => {
   return states.length === 0 ? 'Trap' : states.sort((a, b) => a - b).join(', ')
 }
 
+/**
+ * Returns all read symbols for a transition
+ */
+const readSymbols = (t: FSAAutomataTransition): string[] => {
+  // Either split all the character, or return a lambda transition
+  return t.read ? t.read.split('') : ['']
```

Comment:
Do you need to chuck `expandReadSymbols(t.read)` here for read strings like `[a-z]`?

Reply:
Oh shoot forgot about that.
Think that function might handle everything actually